java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/RoutingController.java

```diff
@@ -161,11 +177,15 @@ public RoutingParameter(String key,  String name, String description, String gro
 	@RequestMapping(path = "/routing-modes", produces = {MediaType.APPLICATION_JSON_VALUE})
 	public ResponseEntity<?> routingParams() {
 		Map<String, RoutingMode> routers = new LinkedHashMap<>();
-		RoutingParameter nativeRouting = new RoutingParameter("nativerouting", "Development", 
-				"[Dev] C++ routing", true);
+		RoutingParameter hhRouting = new RoutingParameter("hhrouting", "Development",
+				"[Dev] Use HH (auto)", true);
+		RoutingParameter hhOnly = new RoutingParameter("hhonly", "Development",
+				"[Dev] Use HH (only)", false);
+		RoutingParameter nativeRouting = new RoutingParameter("nativerouting", "Development",
+				"[Dev] Use C++ (routing)", true);
```

Comment:
default false

Reply:
If we set it to default false now, we might overload our server by profile=pedestrian requests !

Reply:
? profile=pedestrian - don't understand. [Dev] Use C++ (routing) - default false

Reply:
9f8dc4f0c565a20b9232ab11e89018c910865e69